docs/competition/teams/photo-studio.md

```diff
@@ -0,0 +1,21 @@
+# Photo studio manager
+
+The role of the photo studio manager is to take nice photos of teams with their robots.
```

Comment:
we also want a photo of the robot on its own

Reply:
Fixed in cf7321e